doc/api/http.md

```diff
@@ -563,11 +563,15 @@ See also: [`request.setTimeout()`][].
 ### Event: `'upgrade'`
 <!-- YAML
 added: v0.1.94
+changes:
+  - version: REPLACEME
+    pr-url: https://github.com/nodejs/node/pull/36901
+    description: The `head` argument is deprecated and is always an
```

Comment:
Can you please also add the same entry for the server [`'upgrade'`](https://nodejs.org/api/http.html#http_event_upgrade_1) event?